src/WebSdk/Publish/Tasks/Tasks/TransformWebConfig.cs

```diff
@@ -25,6 +25,15 @@ public class TransformWebConfig : Task
         [Required]
         public bool UseAppHost { get; set; }
 
+        /// <summary>
+        /// The Target Framework Identifier.
+        /// <br>.NET Framework = ".NETFramework"</br>
+        /// <br>.NET Standard = ".NETStandard"</br>
+        /// <br>.NET Core/.NET 5+ = ".NETCoreApp"</br>
+        /// </summary>
+        [Required]
+        public string TargetFrameworkIdentifier { get; set; }
```

Comment:
Adding a new required parameter halfway through the release is risky if the assembly versions of the task assembly hasn't changed.

Reply:
Well the issue with ``IsWebConfigTransformDisabled`` is that then it would no longer transform ``Web.Debug.config`` and ``Web.Release.config`` which is unacceptable and is a security risk when doing a Release config publish and it not removing ``debug`` from ``compilation`` inside of the Web.config on the published copy.

Also this required parameter should be ok as the .NET SDK sets it by default anyways for all supported Target Frameworks in it. The risk should be minimal with that.

Reply:
The targets that run for .NET Framework projects are different from this. Can you please share a sample project (that has the issue) so that I can understand the issue better?

Reply:
Can do.

Reply:
IsWebConfigTransformDisabled  is used by both ASP.NET Core and the full framework projects. Can you try setting the other property `IsTransformWebConfigDisabled` to false. This is only used by ASP.NET Core and it should allow the other web.config transforms to run in the full framework project.

Reply:
When I tried that it looks like it sets ``RunXdt`` to false.

```xml
      <_WebConfigTransformCompleted Condition="'$(_IsAspNetCoreProject)' == 'true'
                                    And '$(IsTransformWebConfigDisabled)' != 'true'
                                    And '$(IsWebConfigTransformDisabled)' != 'true'
                                    And '$(PublishIISAssets)' != 'false'">true</_WebConfigTransformCompleted>
      <_WebConfigTransformCompleted Condition="'$(_WebConfigTransformCompleted)' == ''">false</_WebConfigTransformCompleted>
      <RunXdt Condition="'$(RunXdt)' == ''">$(_WebConfigTransformCompleted)</RunXdt>
```
As such I done a new one 2 hours ago which is ``IsAspNetCoreInWebConfigDisabled`` which would not affect that then.